styles.css

```diff
@@ -4,6 +4,67 @@
   font-family: poppins, sans-serif;
 }
 
+.mobile-menu {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: #6070ff;
+  display: none;
+  flex-direction: column;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  mix-blend-mode: multiply;
+}
+
```

Comment:
Please ensure your mobile menu style matches the given Figma design to create a better UI design. 👍🏽
Figma design | Your Design
![Screenshot from 2023-05-16 11-58-56](https://github.com/marvin-nyalik/NyalikPortfolio/assets/44624138/7a55a02c-09b1-4307-9e82-4d43442b45c6) | ![Screenshot from 2023-05-16 11-52-56](https://github.com/marvin-nyalik/NyalikPortfolio/assets/44624138/95d782cb-be4a-4fe1-8de2-2d93758202cf).

Reply:
I have made the required changes as requested above with my coding partner as can been seen [in this commit](https://github.com/marvin-nyalik/NyalikPortfolio/commit/a3645ee2dd8662978834ed0a9bd91ea84f895c36) and [here as well](https://github.com/marvin-nyalik/NyalikPortfolio/commit/4653d27ffe9b6b77ed3aa8353578220dc37aaac1).

Let me know of any more required changes😊...

Reply:
Well resolved ✔️ 

![image](https://github.com/marvin-nyalik/NyalikPortfolio/assets/38088818/4836064f-2ec3-4991-a72f-1032c5b8eb69)